script/Ciel/Day 12/12_01_CIEL12_1.txt

```diff
@@ -6,32 +6,32 @@
 [sha:5a06c1bf27aa9d0225825279af82ef6183b07ea3]{
 -- Page 1, Offset 30867.
 -- 勢いよくベッドから跳ね起きた。
-I energetically leap out of bed.
+I awaken with vigor, bolting upright in bed.
 }
 [sha:21f5ac19360322e3ceaa517b0e9de53dc9d18eeb]{
 -- Page 2, Offset 30868.
 -- 酷い夢を見ていたのか、体はべっとりと汗をかいていた。
-My body is completely drenched in sweat, perhaps because I was seeing a terrible dream.
+Perhaps I had a bad dream, because my body has been soaked in sweat.
 }
 [sha:9091897d885a48286aa720aaf7ab7667b1c15805]{
 -- Page 2, Offset 30869.
 -- ……けど、うなされるような夢を見た覚えはない。
-...But I don't remember having a nightmare.
+But... I don't remember dreaming of something that would trouble me so in my sleep.
```

Comment:
revert unless there's some specific nuance in the うなされるような夢 i'm missing

Reply:
This sentence was a massive simplification. 
I know うなされる is usually just described in EN-JP dicts as 'having a nightmare' but its more about the behavior that a nightmare causes in someone sleeping: "to toss in one's sleep".
In that sense, the reason why Shiki even brings it up, is because he's trying to rationalize the whole thing about him sweating:
1. Hmm, I'm sweaty
2. But I don't remember dreaming about something that would make me toss around that much that I'd be drenched in that sweat.